src/actions/cycleJobCategories.js

```diff
@@ -4,173 +4,232 @@ import {
   UPDATE_CYCLE_JOB_CATEGORIES_SUCCESS,
   UPDATE_CYCLE_JOB_CATEGORIES_SUCCESS_TITLE,
 } from 'Constants/SystemMessages';
+import { CancelToken } from 'axios';
 import { batch } from 'react-redux';
 import api from '../api';
 import { toastError, toastSuccess } from './toast';
+import { convertQueryToString } from '../utilities';
 
-// =================== CYCLE JOB CATEGORIES EDIT ===================
+let cancelCycleCategories;
+let cancelCycleJobCategories;
+let cancelCycleJobCategoriesStatuses;
+// let cancelEditCycleJobCategories;
 
-export function cycleJobCategoriesEditErrored(bool) {
+// =================== CYCLE CATEGORIES LIST ===================
```

Comment:
```suggestion
// =================== CYCLE CATEGORIES ===================
```
To match reducer comments

This is sooo nit-picky and the only reason i point it out is cause i sense you may appreciate it ❤️